packages/expect/src/jasmine_utils.js

```diff
@@ -222,7 +222,7 @@ function keys(obj, isArray, hasKey) {
   }
 
   for (var x = 0; x < allKeys.length; x++) {
-    if (!allKeys[x].match(/^[0-9]+$/)) {
+    if (typeof allKeys[x] === 'symbol' || !allKeys[x].match(/^[0-9]+$/)) {
```

Comment:
Just add a `FlowFixMe` comment above this, flow is wrong.

![image](https://user-images.githubusercontent.com/1404810/40920056-bf6a83b8-680b-11e8-9831-aca689a666e4.png)

Reply:
Pushed, used `//$FlowFixMe`. Hopefully that is correct, I am flow noob :)